backend/main.py

```diff
@@ -22,24 +22,29 @@
 
 Base = declarative_base()
 
+USER_NEWS_ASSOCIATION_TABLE_NAME = "user_news_upvotes"
+USERS_TABLE_NAME = "users"
+NEWS_ARTICLES_TABLE_NAME = "news_articles"
 
 user_news_association_table = Table(
-    "user_news_upvotes",
+    USER_NEWS_ASSOCIATION_TABLE_NAME,
```

Comment:
The name is located under the Table. I think it is enough to describe its purpose and significance. If you want to provide more information, just use an explicit way to implement the class

`user_news_association_table = Table(name="user_news_upvotes", ...)`